### DISCUSSION TITLE
Any idea why fish shell doesn't autocomplete doctl

### DISCUSSION THREAD

**Discussion by awkto** [fish-shell/fish-shell#10026] [Q&A]
Any idea why fish shell doesn't autocomplete [doctl (DigitalOcean CLI](https://github.com/digitalocean/doctl))

The tool has an autocomplete flag which works in bash. 

But when I switch into fish-shell, the autocomplete doesn't work. Hitting tab after the doctl command just brings up files/folders options from the current directory.

Curious why that's the case, as other CLI tools like terraform's autocompletion work fine in fish-shell without any additional steps

**Comment by faho** [comment]
>Any idea why fish shell doesn't autocomplete [doctl (DigitalOcean CLI](https://github.com/digitalocean/doctl))

Because nobody has hooked it up: There is no doctl.fish in https://github.com/fish-shell/fish-shell/tree/master/share/completions.

Well, no, that's not correct - since https://github.com/fish-shell/fish-shell/commit/e682ffaf11dde439bb6bc77a24e428d029bf3ca8 there is.
